src/category_theory/full_subcategory.lean

```diff
@@ -106,6 +106,16 @@ induced_category.full subtype.val
 instance full_subcategory.faithful : faithful (full_subcategory_inclusion Z) :=
 induced_category.faithful subtype.val
 
+variable {Z}
+
+lemma is_iso_iff_is_iso {X Y : {X : C // Z X}} (f : X ⟶ Y) :
+is_iso f ↔ @is_iso C _ _ _ f :=
```

Comment:
```suggestion
  is_iso f ↔ @is_iso C _ _ _ f :=
```
Should this be flipped around, and made into a simp-lemma?

Reply:
I'm not sure which way should the simp normal form be. In all my use cases (other PRs as well), This is all used to rewrite in this direction. However it may be desirable for simp to rewrite in the other direction?